ui/apps/platform/src/Containers/Vulnerabilities/utils/sortUtils.tsx

```diff
@@ -51,9 +45,9 @@ export function getDefaultWorkloadSortOption(entityTab: WorkloadEntityTab): Sort
         case 'CVE':
             return { field: 'CVSS', aggregateBy: aggregateByCVSS, direction: 'desc' };
         case 'Deployment':
-            return { field: 'Deployment Risk Priority', direction: 'asc' };
+            return { field: 'Deployment', direction: 'asc' };
         case 'Image':
-            return { field: 'Image Risk Priority', direction: 'asc' };
+            return { field: 'Image created time', direction: 'asc' };
```

Comment:
Waiting for final confirmation from PM on what the default sort fields should be, so I will make a ticket to revisit this if needed.